src/main/java/org/usfirst/frc/team4999/controllers/LogitechF310.java

```diff
@@ -125,6 +125,10 @@ public boolean getBackButtonPressed() {
 	public boolean getBackButtonReleased() {
 		return getRawButtonReleased(BACK);
 	}
+
+	public double getArmsSpeed() {
+		return 0;
```

Comment:
So your arms are never  going to move?
Come on, I basically wrote two implementations of this method for you and you can't be bothered to chose one and paste it in?